Android.mk

```diff
@@ -36,8 +36,13 @@ HYBRIS_R_ALWAYSDEBUG := 1
 ## All manual "config" should be done above this line
 
 # Force deferred assignment
-
-HYBRIS_FIXUP_MOUNTS := $(LOCAL_PATH)/fixup-mountpoints
+ifeq ($(shell test -e rpm/fixup-mountpoints && echo -n yes),yes)
+    $(info USE CUSTOM fixup-mountpoints )
+    HYBRIS_FIXUP_MOUNTS := rpm/fixup-mountpoints
+else
+    $(info USE HYBRIS fixup-mountpoints )
+    HYBRIS_FIXUP_MOUNTS := $(LOCAL_PATH)/fixup-mountpoints
+endif
```

Comment:
This would need to be tested on exclusively-OBS builds as they might break if using this functionality. OBS builds are currently not available in public.

Alternatively, a submodule approach could be used for device-specific matters, just like dhd, configs, or droid-hal-version are done. This would mean to re-design hybris-boot to be submodule friendly.